feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go

```diff
@@ -276,6 +282,12 @@ func TestBGP(t *testing.T) {
 			}
 
 			dut := ondatra.DUT(t, "dut")
+
+			t.Log("Configure Network Instance")
```

Comment:
This is supposed to be done inside tc.dut.Configure() if not already.

Reply:
Moved network instance config to  tc.dut.Configure(). Thanks.